nautobot/extras/models/jobs.py

```diff
@@ -469,6 +477,65 @@ class Meta:
         verbose_name_plural = "job log entries"
 
 
+#
+# Job Queues
+#
+
+
+@extras_features(
+    "custom_links",
+    "graphql",
+    "webhooks",
+)
+class JobQueue(PrimaryModel):
+    """
+    A Job Queue represents a structure that is used to manage, organize and schedule jobs for Nautobot workers.
+    """
+
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, unique=True)
+    description = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
+    type = models.CharField(
+        max_length=50,
+        choices=JobQueueTypeChoices,
+        help_text="Job Queue Type, can be one of two types (Celery, Kubernetes)",
```

Comment:
I'm not sure this `help_text` adds much since we already have explicit `choices` defined? It's something that could easily get outdated by accident if/when we add additional choices.